docs/global.md

```diff
@@ -131,6 +132,10 @@ _:blank rdf:type rdf:Statement .
 _:blank rdf:type owl:Axiom .
 ```
 
+## Input Format
+
+When loading an ontology, OWLAPI will attempt to load with a variety of ontology format parsers. When all parsers fail, many error logs are printed (when using verbose mode). Finding the cause of the error based on parser can be difficult due to number of messages. For debugging purposes, you can specify the `--input-format` of the ontology as one of: `owl`, `obo`, `owx`, `ofn`, `omn`, `ttl`, or `json`.
```

Comment:
would be good to update this part of the documentation to have a table with columns:
1. extension (e.g., `owl`, `obo`, `owx`, `ofn`, `omn`, `ttl`, or `json`)
2. human readable name for this format
3. link to documentation about the format

Reply:
Maybe important to make it more clear that these formats aren't actually the extension (which can be anything, even potentially mismatched, e.g., how VariO uses `.owl` for the OFN format; http://www.variationontology.org/vario_download/VariO_1.09.owl)

Reply:
Good idea - thanks for the suggestion.